integration_tests/tests/admin/userManagement.cy.ts

```diff
@@ -335,4 +335,27 @@ context('User management', () => {
     // Then the page should show the results for the second page
     page.shouldShowUsers(usersForResultsPage2)
   })
+
+  it('retrieve the user from api if version not matched to user in session', () => {
```

Comment:
the test description doesn't describe the assertions the tests takes.   
Could we assert: 
- if the user version in the response header is different to the value we have locally then a call to `getActingUser` is made and the local version is updated?